Wireframe/index.html

```diff
@@ -4,10 +4,35 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <main>
+      <section class="toptitle">
+        <h1 class="heading">Understanding GIT</h1>
+        <p class="heading_summary">This page will tell you the basics of GIT and why it is useful.</p>
+      </section>
+      <section class="HERO">
```

Comment:
Why is this capitalised? Why is the subsequent class hero not capitalised? Can you talk through what you are trying to achieve in this code?

Reply:
I wanted to create a grid and I was confused how to do that. But after doing other projects now I figured out how to properly do grid and sorted this out.